vulnerable_people_form/form_pages/nhs_login_callback.py

```diff
@@ -53,4 +53,5 @@ def get_nhs_login_callback():
 
 @form.route("/start", methods=["GET"])
 def get_start():
-    return redirect("/view-or-setup")
+    session.clear()
```

Comment:
should we be clearing the session at the start of the journey is there a problem if a user goes all the way back to this page and has already got some info in the session?

Reply:
The `/start` route redirects the browser with a 302, consequently `/start` isn't placed in the browser's history, so the back button isn't an issue here.

The `/start` route is mostly there to force a new session to be instantiated, and (in my mind) was intended as the URL that the external service would link to, when a user follows a "start the form" link.

The presence and behaviour of this endpoint is something carried over from the legacy Ruby application. It can (and maybe should) be removed. However, that's likely out of scope of this PR.